ipinfo/cmd_init.go

```diff
@@ -160,9 +160,11 @@ func cmdInit() error {
 			return err
 		}
 
-		browser.OpenURL(body.SignupURL)
-		fmt.Println("If the link does not open, please go to this link to get your access token:")
-		fmt.Println("")
+		if gConfig.OpenBrowser {
+			browser.OpenURL(body.SignupURL)
+			fmt.Println("If the link does not open, please go to this link to get your access token:")
```

Comment:
I think it can be misleading as in case of disabled the link shouldn't be even opening so displaying `if the link does not open` might mislead. @UmanShahzad what do you suggest?

Reply:
Most users who will be doing `init` for a signup purpose, will not have put `open_browser=disable` yet, so this flag will likely never come into play.

And even if it did come into play, we should've still printed something to tell the user to open the signup URL that is pasted below.

Let us just ignore this case and not check for it.